tests/test-class-wp-web-app-manifest.php

```diff
@@ -64,6 +64,7 @@ class Test_WP_Web_App_Manifest extends TestCase {
 	public function setUp() {
 		parent::setUp();
 		$this->instance = new WP_Web_App_Manifest();
+		update_option( 'pwa_maskable_icon', true );
```

Comment:
After thinking about this some more, I don't think we should assume the icon is maskable. I had written this in the issue:

> But these both assume that the icon is actually maskable. Is this good to assume?

If we were to default maskable to true, then the option should rather be called something like `site_icon_unmaskable` since it would be empty by default, unless we explicitly set it as part of a database migration. But in any case, I think we should explicitly require the user to click the checkbox to say that it is maskable.